engine/wwtlib/Utilities/Histogram.cs

```diff
@@ -71,6 +71,7 @@ public void Show(Vector2d position)
         public static void UpdateImage(ImageSetLayer isl, double z)
         {
             FitsImage image = isl.ImageSet.WcsImage as FitsImage;
+            if(image == null) { return; }
```

Comment:
Formatting nit here and below, I think it's best to use newlines for such statements to make them more perceptible on a quick skim:

```c#
if (image == null) {
    return;
}
```